lib/bambora/builders/xml_request_body.rb

```diff
@@ -5,15 +5,13 @@ module Builders
     ##
     # Builds an XML request body from a Hash
     class XMLRequestBody
-      attr_reader :body, :response_format
+      attr_reader :body
 
-      def initialize(body:, response_format: nil)
+      def initialize(body:)
         @body = body
-        @response_format = response_format
       end
 
       def to_s
-        body.merge!(rpt_format: response_format) if response_format
```

Comment:
Just curious, why was this needed originally? Was it because we thought we may need to use different return formats, but after going through this, it was realized that we simply want JSON back?

Reply:
I moved it because it wasn't specific to formatting an XML body but should instead be passed into this class.